frontend/challedu-app/src/App.css

```diff
@@ -1,38 +1,39 @@
-.App {
-  text-align: center;
+
+#root{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background: url(../src/images/wallpaper.svg);
+  background-size: cover;
+  font-family:'Segoe UI';
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.welcome{
+  text-align: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+#logo{
+  width: 20%;
+  height: auto;
+  
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+#logo{
+  margin-top: 4%;
+  margin-bottom: 6%;
```

Comment:
use bootstrap classes for margin `mt-5 mb-5`
https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
same thing

Reply:
Same as above